src/timeline-report.js

```diff
@@ -45,11 +46,13 @@ export class TimelineReport extends StacheElement {
         <div id='select-issue-type' class='pt-1'></div>
         
         <select-report-type 
-          jiraHelpers:from="this.jiraHelpers"
-          features:from="this.features"></select-report-type>
-          
+        jiraHelpers:from="this.jiraHelpers"
+        features:from="this.features"></select-report-type>
+        
+        <div id='compare-slider' class='flex-grow px-2'></div>
```

Comment:
How should I go about removing the old canjs slider? Should I just delete this code?

Reply:
Delete that code and the files